vignettes/package_extensions.Rmd

```diff
@@ -0,0 +1,95 @@
+---
+title: "Package Extensions"
+output: 
+  rmarkdown::html_vignette
+    
+vignette: >
+  %\VignetteIndexEntry{Package Extensions}
+  %\VignetteEngine{knitr::rmarkdown}
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+
+link <- function(text, url) {
+  return(
+    paste0(
+      "[", text, "]",
+      "(", url, ")"
+    )
+  )
+}
+
+dyn_link <- function(text,
+                     base_url,
+                     relative_url = "",
+                     # Change to TRUE when admiral adopts multiversion docs
+                     is_multiversion = TRUE,
+                     multiversion_default_ref = "main") {
+  url <- paste(base_url, relative_url, sep = "/")
+  if (is_multiversion) {
+    url <- paste(
+      base_url,
```

Comment:
I thought this stuff was just for the readme?

Reply:
Sorry, you are probably right. As you know the text in this page was originally copied from the admiraltemplate README and I needed this `dyn_link` stuff for it to work - do you know how i should be linking to other pages instead? At the moment if I removed this entirely, calls such as the one on line 63 fail. Thanks.

Reply:
fine to remove this section and update any links mentioned below to just use the main release website links, now that this is in admiraldev is the first in the dependency chain

Reply:
updated links, thank you